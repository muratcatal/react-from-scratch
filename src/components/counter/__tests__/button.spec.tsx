import React from "react";
import Button, { ButtonType } from "../button";
import renderer from "react-test-renderer";

describe("Button component testing", ()=>{
    test("button snapshot testing",()=>{
        const component: any = renderer.create(<Button type={ButtonType.Increment} onClick={()=>{}} />);
        let tree: any = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});