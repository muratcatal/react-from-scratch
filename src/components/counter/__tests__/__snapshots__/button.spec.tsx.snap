// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component testing button snapshot testing 1`] = `
<button
  className="MuiButtonBase-root-16 MuiButton-root-1 MuiButton-flatPrimary-3"
  disabled={false}
  onBlur={[Function]}
  onClick={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex="0"
  type="button"
>
  <span
    className="MuiButton-label-2"
  >
    +
  </span>
  <span
    className="MuiTouchRipple-root-19"
  />
</button>
`;
